### DISCUSSION TITLE
Daily  12/06

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#589] [Daily Sprint]
Con respecto al proyecto:

- ¿Qué hice ayer?
- ¿Qué haré hoy?
- Impedimentos o dificultades

**Comment by rpgrca** [comment]
Ayer hice casi todo lo de ofertas, se pueden crear y asignar productos, todavía no calcula en el carrito el precio con descuento. Tampoco se pueden editar los productos de la oferta pero eso tiene menos prioridad, en todo caso que borre la oferta y vuelva a crearla. Hoy voy a hacer lo de envíos y empezar con lo de la compra del carrito porque lo necesito sí o sí para hacer lo del pago.

**Comment by NataliaAlvarezIspc** [comment]
Entre el viernes y el sábado estuve terminando las tareas que tenía asignadas de frontend. El sábado por la noche tuvimos nuestra weekly. Ayer estuve trabajando con la tarea del buscador para que se muestre en un popup el resultado. No pude revisar y corregir la documentación.
Hoy voy a buscar que tareas puedo realizar y tratar de corregir la documentación.

**Comment by BraianTroncoso** [comment]
Estuve trabajando en las issues asignadas, pude realizar todas las planteadas hasta el momento, solamente estoy renegando con la que ya tuve el finde y no pude. Hoy voy a tratar de resolverla, en caso de no poder con tiempo voy avisar.
